src/aioquic/quic/recovery.py

```diff
@@ -115,17 +119,17 @@ def __init__(
 
         # congestion control
         self._cc = create_congestion_control(
-            congestion_control_algorithm, max_datagram_size=max_datagram_size
+            congestion_control_algorithm, max_datagram_size=max_datagram_size, recovery=self
         )
         self._pacer = QuicPacketPacer(max_datagram_size=max_datagram_size)
 
     @property
     def bytes_in_flight(self) -> int:
-        return self._cc.bytes_in_flight
+        return int(self._cc.bytes_in_flight)
```

Comment:
This is already an int, no need to cast it.

Reply:
> In cubic, this can be a float, leading to some unexpected error, so I just casted it when trying to access it

Uh no I don't see how this can be a float, we start with 0 and all we do is add / subtract packet lengths. This cannot and must not be a float.